src/pages/getting-started/livechat-apps/index.mdx

```diff
@@ -46,6 +42,127 @@ See a 4-minute tutorial explaining how to create a simple app in the Developer C
+6. Click the **Continue button** and enjoy your new app.
+
+<SectionLink to={"/getting-started/app-guides/agent-app-widgets#building-block-configuration"}>
+  Take a look at guidelines on how to fill in Building blocks details.
+</SectionLink>
+
+<img
+  alt="Creating LiveChat apps step2.1"
+  src="/images/getting-started/livechat-apps/LiveChat-appsstep2.1.1.png"
+  width="1515px"
+  height="819px"
+/>
+
+### **Step 3  Configuring Display details**
+
+Now, it is time for the **Display details** tab that allows you to configure your app's short description and add an icon for better visibility in the LiveChat App.
+
+Follow the steps below to add a description and icon:
+1. Go to the **Display details** tab.
+2. In the **Short description** section, add a one-sentence description of your app to make your users aware of what your app does, and save your changes. You can change the description later.
+3. Add an icon for your app and save your changes. Your icon will be displayed on the Marketplace and the LiveChat App. You can drag and drop your image or browse your file manager for it.
+
+<img
+  alt="Creating LiveChat apps step3"
+  src="/images/getting-started/livechat-apps/LiveChat-apps step3.png"
+  width="1515px"
+  height="819px"
+/>
+
+### **Step 4 Configuring Listing details**
+
+The next step is adding a few details of your app’s functionalities to make it easier to understand for your users. 
+
+To do that, go to the **Listing details** tab and fill in all the details.
+
+<SectionLink to={"/monetization/app-review-process/#app-details"}>
+  How to fill in app details.
+</SectionLink>
+
+<img
+  alt="Creating LiveChat apps step4"
+  src="/images/getting-started/livechat-apps/LiveChat-apps step4.png"
+  width="1515px"
+  height="819px"
+/>
+
+### **Step 5 Testing your application **
+
+Before you publish your application on the Marketplace, you need to test it first to ensure that no bugs or errors appear. 
+
+To do that, you need to install your app privately on your LiveChat license by going to the **Private installation** tab and clicking the **Install app** button.
+
+<img
+  alt="Creating LiveChat apps step5"
+  src="/images/getting-started/livechat-apps/LiveChat-apps step5.png"
+  width="1515px"
+  height="819px"
+/>
+
+When the installation is done, you can return to your LiveChat App and refresh the website. After doing that, you’ll see your widget. 
+
+Please remember that your widget placement in the LiveChat App will depend on which widget placement you've chosen while creating the new Agent App Widget.
+
+<SectionLink to={"/monetization/app-review-process/#common-problems"}>
+  Take a look at things to remember when testing the app.
+</SectionLink>
+
+<img
+  alt="Creating LiveChat apps step5.1"
+  src="/images/getting-started/livechat-apps/LiveChat-apps step5.1.png"
```

Comment:
```suggestion
  src="/images/getting-started/livechat-apps/LiveChat-apps-step5.1.png"
```

Reply:
I suggest renaming the image files so they don't include spaces so it's consistent with our other pics names :D I already "changed" the naming in the tutorial so you can just change the file names on your local branch after we're set on the changes here

Reply:
got it :D I did not know that it may have some consequences for the future. I will remember that for the future :D